components/steplist/index.css

```diff
@@ -126,7 +126,9 @@ governing permissions and limitations under the License.
   /* Selected state */
   &.is-selected {
     /* Focus */
-    &:focus {
+    &:focus,
+    &.is-focused,
+    *:focus {
```

Comment:
This is a little curious; you're setting `outline: none` on any child element of a selected step that has focus (i.e. this selector resolves to `.spectrum-Steplist-item.is-selected *:focus`. Was that intended?

Reply:
This is to account for the interactive use case where a link is the child of the `.spectrum-Steplist-item`.

Reply:
https://github.com/adobe/spectrum-css/pull/217/files#diff-90810980a9333ecd68c5d9e338f0e818R40